### ISSUE TITLE
AttributeError when using `uproot.pandas.iterate`.

### ISSUE THREAD

**Issue by marinang** [scikit-hep/uproot#331]
Hi I get the following error when doing

```python
for df in uproot.pandas.iterate(files, "ntuple/DecayTree",
    namedecode="utf-8", branches=branches):
```

```python
File "/afs/cern.ch/work/m/mmarinan/anaconda3/envs/analysis/lib/python3.7/site-packages/uproot/tree.py", line 126, in iterate
    arrays.index._start += globalentrystart
AttributeError: can't set attribute
```

**Comment by jpivarski** [comment]
Does PR #332 solve the issue?

**Comment by marinang** [comment]
It does thanks 👍

**Comment by jpivarski** [comment]
This will be included in uproot 3.10.0, a major release including TTree-writing!
